app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -150,6 +159,16 @@ def remote_execution_proxies(provider, *_rest)
 
         scoped_search relation: :pools, on: :pools_expiring_in_days, ext_method: :find_with_expiring_pools, only_explicit: true
 
+        def add_back_cve_errors
+          if @pending_cve_attrs&.[](:content_view_id).present? || @pending_cve_attrs&.[](:lifecycle_environment_id).present?
```

Comment:
```suggestion
          if @pending_cve_attrs.key?(:content_view_id) || @pending_cve_attrs.key?(:lifecycle_environment_id)
```

Reply:
That is probably not going to work first time when @pending_cve_attrs is nil

Reply:
you're right! but I want to check values, not keys. Updated